homeassistant/components/fan/__init__.py

```diff
@@ -629,7 +630,7 @@ def state_attributes(self) -> dict:
     @property
     def supported_features(self) -> int:
         """Flag supported features."""
-        return 0
+        return self._attr_supported_features
 
```

Comment:
This whole property should be removed, so it's handled from base class.

Reply:
you are right 👍